src/pages/workspace/distanceRates/PolicyDistanceRateDetailsPage.tsx

```diff
@@ -51,7 +51,7 @@ function PolicyDistanceRateDetailsPage({policy, route}: PolicyDistanceRateDetail
     const taxClaimablePercentage = rate.attributes?.taxClaimablePercentage;
     const taxRateExternalID = rate.attributes?.taxRateExternalID;
 
-    const isDistanceTrackTaxEnabled = customUnit?.attributes?.taxEnabled;
+    const isDistanceTrackTaxEnabled = Boolean(customUnit?.attributes?.taxEnabled);
```

Comment:
We should use `!!` 
Boolean is getting deprecated https://github.com/Expensify/App/pull/42492/files

Reply:
I updated, please check again 🙏